### PR TITLE
Bump ember-test-selectors to latest to fix tests and remove ember-cli-htmlbars-precompile

### PR DESCRIPTION
**Pull request by snewcomer**
Build errors in nightly run - https://github.com/simplabs/ember-simple-auth/runs/3529613045?check_suite_focus=true

### CODE REVIEW

**Comment by snewcomer** [comment]
@marcoow @BobrImperator We are getting an Ember global failure.  That add-on is pretty outdated.  Is this something you have access to?  Basically I don't know how we can get beta passing (with no Ember global usage) without ensuring all dependencies are up to date.

```
WARNING: [DEPRECATION] [DEPRECATION] Usage of the Ember Global is deprecated. You should import the Ember module or the specific API instead.

* Upgrade the following addons to the latest version:
  * torii
```

**Comment by BobrImperator** [comment]
@snewcomer I don't think there's a way to get ember-beta passing while global Ember is still used it's usage was expected to be removed in 4.0 https://deprecations.emberjs.com/v3.x#toc_ember-global.

Torii unfortunately as you've probably noticed had gone silent a few years ago, looks like there's plan to move it to Adopted Ember Addons.
https://github.com/Vestorly/torii/issues/461

**Comment by snewcomer** [comment]
Been an issue on a few projects now 😅.  How do you want to proceed?  Are you ok with the beta failures for now to get this PR and #2317 in?

**Comment by BobrImperator** [comment]
Definitely, to me it looks alright :+1: I'd personally even consider disabling ember-beta for now and meanwhile keep working on the blocking deprecations. This is unfortunately a community wide issue right now  which will likely be going for next few months.

@marcoow

**Comment by snewcomer** [comment]
@BobrImperator @marcoow Since we know these ember-beta tests are failing, is it possible we could move this PR plus these other PRs across the finish line?

https://github.com/simplabs/ember-simple-auth/pull/2320
https://github.com/simplabs/ember-simple-auth/pull/2317

File: packages/ember-simple-auth/package.json [hunk 1]
```diff
@@ -43,7 +43,6 @@
     "ember-cli-content-security-policy": "~1.1.1",
     "ember-cli-dependency-checker": "^3.2.0",
     "ember-cli-htmlbars": "^5.0.0",
-    "ember-cli-htmlbars-inline-precompile": "^3.0.1",
```
Comment (BobrImperator) [inline packages/ember-simple-auth/package.json:46]: Is removing this package necessary to upgrade ember-test-selectors? Could we do it separately instead?

File: packages/ember-simple-auth/package.json [hunk 2]
```diff
@@ -25,7 +25,7 @@
     "broccoli-file-creator": "^2.0.0",
     "broccoli-funnel": "^1.2.0 || ^2.0.0",
     "broccoli-merge-trees": "^2.0.0 || ^3.0.0",
-    "ember-cli-babel": "^7.20.5",
+    "ember-cli-babel": "^7.26.6",
```
Comment (BobrImperator) [inline packages/ember-simple-auth/package.json:28]: Is the bump necessary here? If not could we do it as a separate PR instead?

**Review by BobrImperator — CHANGES_REQUESTED** [review]
Hi @snewcomer sorry for the hold up on this, please see the comments.

If possible, I think it'd be better to do upgrade/remove packages separately unless it's required to bump test-selectors as well.

File: packages/ember-simple-auth/package.json (hunk 1 above)
Comment (snewcomer) [reply packages/ember-simple-auth/package.json:46]: This is a warning in the console saying to remove it.

Do you plan on doing a minor bump for every PR?  If not, I would like to avoid the cycle of PR, wait time for a review, merge, and then back again.

Comment (BobrImperator) [reply packages/ember-simple-auth/package.json:46]: No, I don't plan to release each end every PR separately. It's just better for changelogs to have things separated I think.

**Review by BobrImperator — APPROVED** [review]
Thanks, could you please change PR title to say something like `Update ember-test-selectors to 6` and maybe mention `ember-cli-htmlbars-inline-precompile`?

**Comment by BobrImperator** [comment]
Hi @snewcomer, we've had a meeting today and we'd really like to have `ember-test-selectors` bump and `ember-cli-htmlbars-inline-precompile` removal as separate PRs.

**Review by BobrImperator — CHANGES_REQUESTED** [review]
Hi @snewcomer, we've had a meeting today and we'd really like to have ember-test-selectors bump and ember-cli-htmlbars-inline-precompile removal as separate PRs.
